README.md

```diff
@@ -21,6 +21,7 @@ Questions? Ask them in the [forum](https://groups.google.com/forum/#!forum/opens
 
 * Java 1.7.0+
 * Node/NPM
+* Python
```

Comment:
Looks like you've got a lot of changes that are unrelated in this PR. Maybe a bad merge or rebase?